src/toM3u8.js

```diff
@@ -144,22 +146,17 @@ export const organizeAudioPlaylists = (playlists, sidxMapping = {}) => {
     }
 
     // skip if we already have the highest quality audio for a language
-    if (a[label] &&
-      a[label].playlists[0].attributes.BANDWIDTH >
-      playlist.attributes.bandwidth) {
-      return a;
+    if (!a[label]) {
```

Comment:
This is the fix here, we should create the group, and add playlists to it. Not only create a group with one playlist ever.